### PR TITLE
Store usager instead of sub in connections

### PR DESCRIPTION
**Pull request by thimy**
This PR directly replaces `sub_usager` in the Connection model by a Usager instance.

### CODE REVIEW

File: aidants_connect_web/migrations/0003_auto_20190723_1508.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Generated by Django 2.2.3 on 2019-07-23 13:08
+
+from django.db import migrations, models
```
Comment (Anna-Livia) [inline aidants_connect_web/migrations/0003_auto_20190723_1508.py]: ⚠️ Don't forget to rebase on master and replay the migrations

**Comment by Anna-Livia** [comment]
Great work 🌮 🍰   !!!
Don't forget to rebase and re-run the migrations before merging :)

File: aidants_connect_web/migrations/0003_auto_20190723_1508.py (hunk 1 above)
Comment (thimy) [reply aidants_connect_web/migrations/0003_auto_20190723_1508.py]: This branch seems up to date to me, what am I missing?

Comment (Anna-Livia) [reply aidants_connect_web/migrations/0003_auto_20190723_1508.py]: I might be a bit paranoid about the migrations, but if you are sure everything is up to date with master, then you can merge :)

Comment (thimy) [reply aidants_connect_web/migrations/0003_auto_20190723_1508.py]: Made some cleaning, it seemed that some migration files were left although they were squashed in another file. Is it good with you?

Comment (Anna-Livia) [reply aidants_connect_web/migrations/0003_auto_20190723_1508.py]: The squashed files took the files from migration `0001` to `0003` and squashed them in `0001_squashed_0003_auto_20190725_1048.py`
The squashed files should be kept until `integ` has been updated.

Then, we can delete the squashed migration files.

1. before merging, I think you should update `integ` with the latest master
2. I think you can discard `0001_initial.py` along with the other squashed migrations

**Pull request merged**